### PR TITLE
Bug 1093755 - Closing search when opening utility tray r=kgrandon +autolander

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[albertopq albertopq (albertopq)](https://github.com/albertopq) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0fed698c1047)

**Comment by try-server-hook** [comment]
[albertopq albertopq (albertopq)](https://github.com/albertopq) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ad54c35f0609)

File: apps/system/style/statusbar/statusbar.css [hunk 1]
```diff
@@ -6,7 +6,8 @@
   height: 2rem;
 }
 
-#screen.rocketbar-focused .gesture-panel {
+#screen.rocketbar-focused #left-panel,
```
Comment (KevinGrandon) [inline apps/system/style/statusbar/statusbar.css]: Can you add a quick comment here? Something about that this is for allowing the utility tray to open, but not edge gestures?

**Comment by try-server-hook** [comment]
[albertopq albertopq (albertopq)](https://github.com/albertopq) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=14613d349e03)

**Pull request merged**